fighthealthinsurance/common_view_logic.py

```
@@ -1761,7 +1765,12 @@ async def sub_in_appeals(appeal: dict[str, str]) -> dict[str, str]:
             ):
                 procedure = denial.procedure
             subs = {
+                "Esteemed Members of the Appeals Committee": "insurance_company",
+                "[insurance_company]": insurance_company,
+                "{insurance_company}": insurance_company,
                 "insurance_company": insurance_company,
+                "{insurance_company}": insurance_company,
```

Ruff (0.8.2)

Dictionary key literal `"{insurance_company}"` repeated

Remove repeated key literal `"{insurance_company}"`

(F601)